recipes/normi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "normi" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/normi-{{ version }}.tar.gz
+  sha256: 3e9d815f3332377baf038c36aae16c64d01530f3d3f41c7bf198eac39fc1ad31
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - normi = normi.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.21.0
+    - scipy
+    - click >=8.0.0
+    - beartype >=0.10.4
+    - tqdm
+    - scikit-learn >=1.0.0
+    - typing-extensions >=3.9.0  # [py<39]
```

Comment:
You can remove the selector here, installing typing-extensions in py>=39 is safe b/c it behaves as an empty package and will allow us to keep this noarch.